backend/src/main/java/de/neuefische/backend/service/DoctorService.java

```diff
@@ -50,4 +55,47 @@ public Doctor addDoctor(DoctorDto doctorDto) {
         }
     }
 
+
+    public Doctor addAppointment(DoctorDto doctorDto) {
```

Comment:
this method is way to long try and refactor it into smaller portions that'll make it easier to read/understand